src/core/less/structure/article.less

```diff
@@ -0,0 +1,53 @@
+// Article styling
+// ===============
+
+.article {
+    background-color: @background-color;
+
+    .article-inner {
+        padding: @article-padding-top 0 @article-padding-bottom;
+        @media all and (max-width: (@device-width-large + 20px)) {
```

Comment:
all of the media queries like the ones in this file are additional styling for a specific theme specific look, and should be removed

Reply:
Done.